walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
Is this a predefined function? If not, I would expect to see it in this PR.

Reply:
Oh I see that is is predefined. Follow-up is that we don't need to do PR to modify anything under `walkthroughs/`. Let's reserve the PR process for your portfolio.

Reply:
Yes, its a predefined function in script.js that the walkthrough told us to try out and use.